crates/uv-toolchain/src/downloads.rs

```diff
@@ -0,0 +1,55 @@
+#[derive(Debug, PartialEq)]
+pub struct PythonDownloadMetadata {
+    name: ImplementationName,
+    arch: Arch,
+    os: Os,
+    libc: Libc,
+    major: i64,
+    minor: i64,
+    patch: i64,
+    url: &'static str,
+    sha256: Option<&'static str>,
+}
+
+#[derive(Debug, PartialEq)]
+pub(crate) enum Arch {
+    Arm64,
+    I686,
+    Ppc64Le,
+    S390X,
+    Windows,
```

Comment:
windows?

Reply:
Maybe there's a parsing problem with the "windows-shared" builds.